decidim-proposals/spec/shared/manage_proposals_examples.rb

```diff
@@ -416,6 +416,27 @@
     end
   end
 
+  context "when admin can create proposal notes" do
```

Comment:
There's already this file with system specs:

https://github.com/decidim/decidim/blob/master/decidim-proposals/spec/system/index_proposal_notes_spec.rb

It was not detecting the error because it only checks for "successfully", that's why the tests didn't fail. I'd rather have the current file instead of these tests, although the tests descriptions in the file can be improved